src/utils/review.ts

```diff
@@ -0,0 +1,42 @@
+import { AdminReview } from "@/services/api/admin/useReviews";
+
+function isGreaterThan24Hours(date: Date): boolean | null {
+  if (!date) return null;
+
+  const createdReview = new Date(date);
+  const now = new Date();
+  const differenceInMillis = now.getTime() - createdReview.getTime();
+
+  const millisecondsIn24Hours = 24 * 60 * 60 * 1000;
+
+  return differenceInMillis > millisecondsIn24Hours;
+}
+
+export const getReviewStatus = (review: AdminReview) => {
+  // checks if it is a review, NB: it is used in  the tableItems file
+  if (!review?.transcript?.status) {
+    return false;
+  }
+
+  const isExpired =
+    !review.mergedAt &&
+    review.archivedAt &&
+    (isGreaterThan24Hours(review.createdAt) ||
+      review.transcript.status?.toLowerCase() === "queued");
+
+  if (review.mergedAt) {
+    return "Merged";
+  }
+  if (
+    review.transcript.status?.toLowerCase() === "not queued" &&
+    !review.mergedAt &&
+    review.pr_url
+  ) {
+    return "Pending";
+  }
+  if (isExpired) {
+    return "Expired";
+  } else {
+    return "Active";
+  }
```

Comment:
This else statement is attached to the `if (isExpired)` so that block of code only runs when `isExpired` is falsy. Which is not what was the intention I guess. When you do a bunch of `if`'s with some early return just simply returning at the end of the block of code where the `if`'s are is equivalent to an else block


```suggestion
  if (isExpired) {
    return "Expired";
  } 
  return "Active";
```